example/src/Examples/Graphs/Interpolation.tsx

```diff
@@ -45,6 +45,8 @@ export const Interpolation: React.FC<GraphProps> = ({ height, width }) => {
           strokeJoin="round"
           strokeCap="round"
           color="#cccc66"
+          start={0}
```

Comment:
```suggestion
```

Reply:
this suggested change is not doing what you think it does.